var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -414,7 +421,8 @@ def ensure_parallel_compiler_wrappers(self):
         if self.spec.satisfies('@1.8.21:1.8.22,1.10.2:1.10.7,1.12.0+mpi'):
             with working_dir(self.prefix.bin):
                 # No try/except here, fix the condition above instead:
-                symlink('h5cc', 'h5pcc')
+                if sys.platform != 'win32':
+                    symlink('h5cc', 'h5pcc')
```

Comment:
(question) https://github.com/spack/spack/pull/27021 added some logic for `symlink` analogs on Windows: does this still need to be skipped on Windows?

If so, the following `if` block also calls `symlink` and should also probably change in that case.